tests/unit/models/operation-test.js

```diff
@@ -4,22 +4,13 @@ import {
 } from 'ember-qunit';
 import { stubRequest } from "../../helpers/fake-server";
 import Ember from 'ember';
+import modelDeps from '../../support/common-model-dependencies';
```

Comment:
Might want to refactor these to be called `stackChildModelDeps` or something later. `common` implies common to all models, and these are really constrained to children of stack.